src/api/types/updateSolutionMutation.js

```diff
@@ -0,0 +1,37 @@
+import { GraphQLNonNull, GraphQLString } from 'graphql'
+import { mutationWithClientMutationId, fromGlobalId } from 'graphql-relay'
+import { solutionType } from './solutionType'
+import { ThemeSolution } from '../../database.js'
+
+export const updateSolutionMutation = mutationWithClientMutationId({
+  name: 'updateSolution',
+  inputFields: {
+    solutionId: {
+      type: new GraphQLNonNull(GraphQLString)
+    },
+    solutionData: {
+      type: new GraphQLNonNull(GraphQLString)
+    }
+  },
+  outputFields: {
+    solution: {
+      type: solutionType,
+      resolve: (payload) => {
+        return payload
+      }
+    }
+  },
+  mutateAndGetPayload: ({ solutionId, solutionData }) => {
+    const { id } = fromGlobalId(solutionId)
+    return ThemeSolution.find({
```

Comment:
You can use `ThemeSolution.update(..` directly here

http://docs.sequelizejs.com/manual/tutorial/instances.html#working-in-bulk-creating-updating-and-destroying-multiple-rows-at-once-